### ISSUE TITLE
[Spotter] It does not have in window help nor tool tips.

### ISSUE THREAD

**Issue by tesonep** [pharo-spec/NewTools#192]

**Comment by guillep** [comment]
In the old spotter it was this:

![image](https://user-images.githubusercontent.com/708322/112836637-2335ef80-909b-11eb-9f1d-2ef4c7676ef2.png)

And we did a rolling set of tips that changed when you clicked on it

![image](https://user-images.githubusercontent.com/708322/112836682-30eb7500-909b-11eb-98ae-1a9b9327bd2b.png)

![image](https://user-images.githubusercontent.com/708322/112836708-36e15600-909b-11eb-91a0-ce06cc145e57.png)

**Comment by estebanlm** [comment]
if I am not wrong, the tips where added by @tesonep because he needed to concentrate the results in the just one tag (Results) instead the old Classes, Implementors, etc., then, the information displayed in the category header (which included the hint of adding hashtags and the keybindings) was lost. 
So, this was AFAIK a workaround. 
Now, I restored the categories and the information:  

![image](https://user-images.githubusercontent.com/513630/112965116-415a2900-9149-11eb-8b0b-0c1b6a77fbe2.png)

So, even if I implemented the tips, I didn't see any real reason to include it in the final version.
